README.md

```diff
@@ -13,15 +17,16 @@ For more info on how we tackle the challenge, have a look at our [wiki](https://
 
 ### What we are going to make
 
-Crushers is dedicated to prepare you to **crush the test**, the drivers license test that is. We are going to do so by providing an app which quizes you on the traffic laws, signs, and scenarios. Our app will provide you with explanations, visual illustrations, and a hands on manual driving mode to give you the full experience and prepare you like you have never before been prepared for a test.
+Crushers is dedicated to prepare you to **crush the test**, the theoretical drivers license test that is. We are going to do so by providing an app which quizes you on the traffic laws, signs, and scenarios. Our app will provide you with explanations, visual illustrations, and a hands on manual driving mode to give you the full experience and prepare you like you have never before been prepared for a test.
```

Comment:
Not sure if we should limit ourselves to the theoretical exam here, since under why we make it we say the following:
> To date most apps focus on the theory exam, but our goal is to combine the theory with visual illustrations showing why a certain traffic law/sign is necessary and a manual driving mode in a virtual city to also train you for the driving test.